src/frontend/data/stores/useAppData/index.tsx

```diff
@@ -0,0 +1,6 @@
+import create from 'zustand';
+import { appData } from '../../appData';
+
+export const [useAppData, useAppDataApi] = create(set => ({
+  appData,
```

Comment:
We can maybe do `...appData` to remove one nested level ?

Reply:
Other remark, does-it will parse the DOM everytime we want to use this store ?

Reply:
I'm pretty sure it will not be parsed more than one time.

But I'm just thinking: this was put in the main redux store for consistency, but it does not need to be a store here as it never changes. We could just import `appData` whenever we need it